components/panel/src/PanelHeader.js

```diff
@@ -21,7 +21,7 @@ export const PanelHeader = props => {
 
 	const lookMap = {
 		hero: {
-			color: '#fff',
+			color: BRAND === 'STG' ? COLORS.text : '#fff',
```

Comment:
That's the sort of thing that will now go away :)
So that work was for nothing but good to take note of it to not forget it in the brand STG package

Reply:
Did that before you built the new approach